src/webui/components/notification/elementary/RatedReaction.tsx

```diff
@@ -1,5 +1,15 @@
 
-export default function Component()
+import { AggregatedData as ReactionView } from '^/domain/reaction/aggregate/types';
+import RatedComicReaction from '../elementary/RatedComicReaction';
+import RatedCommentReaction from '../elementary/RatedCommentReaction';
+
+type Props = {
+    readonly reaction: ReactionView;
+};
+
+export default function Component({ reaction }: Props)
 {
-    return <></>;
+    return reaction.comic !== undefined
+        ? RatedComicReaction({ reaction })
+        : RatedCommentReaction({ reaction });
```

Comment:
This works, but is inconsistent with the other components. We currently use the JSX notiation.

```suggestion
        ? <RatedComicReaction comicDataUrl={reaction.comic?.image.dataUrl as string} />
        : <RatedCommentReaction comment={reaction.comment?.message as string} />;
```